ChaosMod/Effects/db/Peds/PedsEternalScreams.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+	Effect by Joshuax8
+*/
+
+#include <stdafx.h>
+
+static void OnTick()
+{
+	for (Ped ped : GetAllPeds())
+	{
+		PLAY_PAIN(ped, 8, 0, 0);
+	}
+}
+
+static void OnStop()
+{
+	for (Ped ped : GetAllPeds())
+	{
+		DISABLE_PED_PAIN_AUDIO(ped, true);
```

Comment:
Any reason to completely disable any pain sounds for every ped on effect end?

Reply:
Actually just noticed it isn't registered in the `RegisterEffect` declaration so it's definitely redundant